cves/kernel/CVE-2016-3135.yml

```diff
@@ -324,8 +330,10 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: | 
+    The commits reference Google's advice and there are sign-offs
+    and reviewers.
```

Comment:
I think this part could benefit from mentioning the reviewers names.

Reply:
Names have now been added.